example/4_objects.html

```diff
@@ -70,7 +72,7 @@
 );
 ground.rotation.x = THREE.Math.degToRad( -90 );
 scene.add( ground );
-scene.add( new THREE.WireframeHelper( ground ) );
+// scene.add( new THREE.WireframeHelper( ground ) );

```

Comment:
Could you uncomment for now or
```
var helper = new LineSegments(
  new WireframeGeometry( ground.geometry ),
  new LineBasicMaterial( { color: 0xffffff } )
);
scene.add( helper );
```

Reply:
Yeah I just haven't had time to go through and make the wireframes for the original objects!